mozilla_django_oidc/auth.py

```diff
@@ -123,52 +153,52 @@ def authenticate(self, **kwargs):
         }
 
         # Get the token
-        response = requests.post(self.OIDC_OP_TOKEN_ENDPOINT,
-                                 data=token_payload,
-                                 verify=import_from_settings('OIDC_VERIFY_SSL', True))
-        response.raise_for_status()
+        token_response = self.get_token_response(token_payload)
```

Comment:
Ditto on the word "response". It's a "token" or "token_info" or "token_object". Perhaps it's just me who thinks "oh it must have a .status_code and .json() if it's a response."

Reply:
done